ballerina/build.gradle

```diff
@@ -219,6 +219,29 @@ publishing {
     }
 }
 
+task startLdapServer() {
+    doLast {
+        if (!Os.isFamily(Os.FAMILY_WINDOWS)) {
+            def stdOut = new ByteArrayOutputStream()
+            exec {
+                commandLine 'sh', '-c', "docker ps --filter name=openldap-server"
+                standardOutput = stdOut
+            }
+            if (!stdOut.toString().contains("openldap-server")) {
+                println "Starting LDAP server."
+                exec {
+                    commandLine 'sh', '-c', "docker-compose -f tests/resources/openldap/docker-compose.yml up -d"
+                    standardOutput = stdOut
+                }
+                println stdOut.toString()
+                sleep(5 * 1000)
+            } else {
+                println "LDAP server is already running."
+            }
+        }
+    }
+}
+
 build.outputs.dir artifactLibParent // To pack Ballerina gRPC CLI
```

Comment:
we need to remove this right?

Reply:
Removed in [c3bb8f0](https://github.com/ballerina-platform/module-ballerina-grpc/pull/1122/commits/c3bb8f086c6c26131311d2ad823467f3bfcb4275)